INSTALL.rst

```diff
@@ -304,13 +304,52 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+Running the network server:
+---------------------------
 
-  haas serve_networks &
 
-as the HaaS user. To make this happen on boot, add the following to ``/etc/rc.local``::
+A systemd script for running the network server is available in the 'scripts' directory.
+Name of the script is: haas_network.service
+
+Centos:
+------
+
+Centos uses systemd to controll all its processes. 
+
+Place the file haas_network.service under:
+``/usr/lib/systemd/system/``
+
+Ubuntu:
+-------
+LTS version of Ubuntu, Ubuntu 14.04 does not come with systemd pre-installed.
+It uses "Upstart" an equivalent of systemd to manage its daemons/processes.
+
+Systemd is available from Ubuntu 15.04 onwards and LTS version 16.04 will ship with systemd by default.
+
+If you are using Ubuntu with any version prior to 15.04, you will have to install systemd before your can use the scripts provided here.
```

Comment:
But I thought it would be fair if I provide that option with a fair warning. 
Also, from the next LTS version systemd is going to be the default for managing daemons/processes. 

To address your concern I can append the old instruction for them. Thus giving the user the choice. Let me know what you think.